openpype/hosts/maya/plugins/publish/collect_render.py

```diff
@@ -354,8 +315,12 @@ def process(self, context):
                 "aovSeparator": aov_separator
             }
 
-            if deadline_url:
-                data["deadlineUrl"] = deadline_url
+            # Collect Deadline url if Deadline module is enabled
+            deadline_settings = (
+                context.data["system_settings"]["modules"]["deadline"]
+            )
+            if deadline_settings["enabled"]:
+                data["deadlineUrl"] = render_instance.data.get("deadlineUrl")
```

Comment:
It might be nicer if this doesn't need to check whether it was enabled but only whether it was collected, right?
```suggestion
            if render_instance.data.get("deadlineUrl"):
                data["deadlineUrl"] = render_instance.data["deadlineUrl"]
```

Or is `deadlineUrl` even collected when not Deadline module is not enabled?

Reply:
`deadlineUrl` is collected by plugin in Deadline module, so yes. But I would add a comment so it is obvious as data in instances and context are hard to track.